-------------------------------------------------------------------------------
Test set: runner.TestRunner
-------------------------------------------------------------------------------
Tests run: 22, Failures: 2, Errors: 0, Skipped: 0, Time elapsed: 50.198 sec <<< FAILURE! - in runner.TestRunner
Then I should see an error message(Scenario: login using invalid credentials)  Time elapsed: 0.006 sec  <<< FAILURE!
org.junit.ComparisonFailure: expected:<[Please check your username and password. If you still can't log in, contact your Salesforce administrator.]> but was:<[]>
	at org.junit.Assert.assertEquals(Assert.java:115)
	at org.junit.Assert.assertEquals(Assert.java:144)
	at stepDefinition.LoginTest.i_should_see_an_error_message(LoginTest.java:60)
	at ✽.Then I should see an error message(feature/Login.feature:18)

Scenario: login using invalid credentials  Time elapsed: 0.006 sec  <<< FAILURE!
org.junit.ComparisonFailure: expected:<[Please check your username and password. If you still can't log in, contact your Salesforce administrator.]> but was:<[]>
	at org.junit.Assert.assertEquals(Assert.java:115)
	at org.junit.Assert.assertEquals(Assert.java:144)
	at stepDefinition.LoginTest.i_should_see_an_error_message(LoginTest.java:60)
	at ✽.Then I should see an error message(feature/Login.feature:18)

